Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/README.md

```diff
@@ -1597,3 +1599,112 @@ For more details about the authentication used in this integration, see <a href=
 </div>
 </div>
 </div>
+<div class="cl-preview-section">
+<h3 id="user-account-disable">13. Disable a user account</h3>
+</div>
+<div class="cl-preview-section"><hr></div>
+<div class="cl-preview-section">
+<p>Disables a user from all Office 365 applications, and prevents sign in. Note: This command disables user, but does not terminate an existing session. Supported only in a self deployed app flow with the Permission: Directory.AccessAsUser.All(Delegated)</p>
```

Comment:
why does it only work for self-deployed apps?
Does this command is also supported only in a self deployed app?

Reply:
The changepassword e.g, is supported only with delegated permissions and not with Application permissions: https://docs.microsoft.com/en-us/graph/api/user-changepassword?view=graph-rest-1.0&tabs=http
IIRC, the cortex XSOAR application auth flow was not supporting delegated permissions at that time [self deployed delegated the ](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#cortex-xsoar-application).

revoke session seems to be supported in both: https://docs.microsoft.com/en-us/graph/api/user-revokesigninsessions?view=graph-rest-1.0&tabs=http

So this can be updated. I suggest putting in the readme the links of the graph API, as this is a sore point for both customers and our support team.